src/xcode/ENA/ENA/Source/Scenes/Checkin/CheckInDetail/TraceLocationDetailViewController.swift

```diff
@@ -0,0 +1,183 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class TraceLocationDetailViewController: UIViewController {
+
+	// MARK: - Init
+
+	init(
+		_ traceLocation: TraceLocation,
+		dismiss: @escaping () -> Void,
+		presentCheckins: @escaping () -> Void
```

Comment:
As we discussed, I think you can remove this.

Reply:
yeah i left it as we need to reload the checkin tableViewController to fetch the new checkin as well

Reply:
The reload is triggered from the eventStore as soon as you add the check-in, using a publisher. No need to tell it to update from your view controller :)